zap/src/main/java/org/zaproxy/zap/extension/alert/AlertViewPanel.java

```diff
@@ -333,6 +341,40 @@ public void keyPressed(java.awt.event.KeyEvent evt) {
                         }
                     });
 
+            AlertTagsTableModel alertTagsTableModel = new AlertTagsTableModel(editable);
+            alertTags = new ZapTable(alertTagsTableModel);
```

Comment:
I know it's more work but it would be more consistent if it followed the other UI components that manage multiple elements (e.g. buttons to allow to add, edit, and remove). It's not clear how one removes a tag.

Reply:
(It could existing classes though.)